src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/naming/illegalidentifiername/InputIllegalIdentifierNamePatternMatchingForSwitch.java

```diff
@@ -0,0 +1,35 @@
+/*
+IllegalIdentifierName
+format = (default)(?i)^(?!(record|yield|var|permits|sealed|when)$).+$
+tokens = (default)CLASS_DEF, INTERFACE_DEF, ENUM_DEF, ANNOTATION_DEF, ANNOTATION_FIELD_DEF, \
```

Comment:
Why is this check, `IllegalIdentifierName` being modified in this PR for `RedundantModifier`?

Reply:
Removed.
probably a leakage between branches (check out to this branch without committing this file in its branch)